pyproject.toml

```diff
@@ -2,6 +2,7 @@
 name = "crds"
 description = "Calibration Reference Data System,  HST/JWST/Roman reference file management"
 authors = [{ name = "STScI CRDS s/w developers" }]
+requires-python = ">=3.11"
```

Comment:
```suggestion
requires-python = ">=3.9"
```

Since we support 3.9+

Reply:
At the moment I can't get numpy 2.0 to be picked up without being in python 3.11.

Reply:
These tests are not even picking up `numpy 2.0`. The requirements file included in this PR does not appear to be doing anything.

Note that if you look at the [dev wheels archive](https://pypi.anaconda.org/scientific-python-nightly-wheels/simple/numpy/) you are linking to in it. There very much are wheels for 3.9 and 3.10. Indeed this is true for both numpy and scipy.

Reply:
Right I found that numpy 2.0 wasn't getting picked up after some digging. Looking into that.